storage/engine/rocksdb.go

```diff
@@ -94,13 +101,19 @@ func (r *RocksDB) Start() error {
 		}
 		return err
 	}
+	r.refcount++
```

Comment:
Use atomic.AddInt32() here too (it shouldn't really matter here but I've seen the race detector pick up on some weird edge cases for things like this)